applications/ParticleMechanicsApplication/custom_utilities/mpm_search_element_utility.h

```diff
@@ -0,0 +1,124 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ \.
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Bodhinanda Chandra
+//
+
+
+#ifndef KRATOS_MPM_SEARCH_ELEMENT_UTILITY
+#define KRATOS_MPM_SEARCH_ELEMENT_UTILITY
+
+// System includes
+#include <cmath>
+#include <set>
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+#include "includes/variables.h"
+#include "utilities/math_utils.h"
+#include "utilities/binbased_fast_point_locator.h"
+#include "particle_mechanics_application_variables.h"
+
+namespace Kratos
+{
+      class MPMSearchElementUtility
```

Comment:
Can you separate this in header & source pls?

Also the class is not needed, it can be in a namespace